dataikuapi/dssclient.py

```diff
@@ -760,18 +760,27 @@ def get_global_usage_summary(self, with_per_project=False):
     ########################################################
 
     def get_variables(self):
+        """
+        Deprecated. Use get_global_variables
```

Comment:
Should we add a method link to the new method for the doc (even if it's right next to it)? Same with deprecated comment in `set_variables`